src/blocks/OrderStatusBlock/Component.tsx

```
@@ -25,7 +25,7 @@ const currentOrders = [
 
 const payload = await getPayload({ config })
 
-interface OrderStatusBlockProps {}
+interface OrderStatusBlockProps { }
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)